### ISSUE TITLE
Random starts on the ground floor

### ISSUE THREAD

**Issue by tinyhoot** [tinyhoot/SubnauticaRandomiser#41]
Long term, it might be nice to have a way to start on the ground floor, rather than always floating on top. That would tie nicely into your "building up" idea, which sounds really interesting!

_Originally posted by @tinyhoot in https://github.com/tinyhoot/SubnauticaRandomiser/issues/40#issuecomment-1221478400_

**Comment by RayRobertson72** [comment]
Should have commented in this issue instead as you probably don't watch closed issues.

https://github.com/tinyhoot/SubnauticaRandomiser/issues/40#issuecomment-1374558266

**Comment by tinyhoot** [comment]
Ah, thanks for the bump. I agree, if this becomes a thing then the whole concept of what counts as "early game" resources needs to be reworked. It would require a logic that is less linear than the current depth-reliant one; perhaps something that considers individual biomes?

**Comment by RayRobertson72** [comment]
> Ah, thanks for the bump. I agree, if this becomes a thing then the whole concept of what counts as "early game" resources needs to be reworked. It would require a logic that is less linear than the current depth-reliant one; perhaps something that considers individual biomes?
> 
> Switching the logic to biomes instead of depth would also have the neat side effect of making it easier to port to BZ, which is generally way less reliant on depth calculations. But that's for a very far-off future.

Yes. It is horrible being stuck 350m down in bulb zone without the mats available to make tanks or fins or any bladder fish, so you can reach the surface to swim to the shallows. That's why I use Accelerated Start to add some tanks to the reach the surface in such cases. I only take them from the lifepod storage if there is no alternative.

Most games with the current Randomiser means you start wherever and swim to the shallows. Adding this would be an awesome alternative twist. Where you don't necessarily have to reach the surface to progress.
